packages/storybook/.storybook/main.js

```diff
@@ -4,9 +4,24 @@ module.exports = {
   stories: ['../@(src|stories)/**/*.stories.@(js|jsx|ts|tsx|mdx)'],
   addons: [
     '@storybook/addon-a11y',
-    '@storybook/addon-essentials',
```

Comment:
addon-essentials is a zero-config for: 
'@storybook/addon-links'
    '@storybook/addon-actions'
    '@storybook/addon-viewport'
'@storybook/addon-docs'
'@storybook/addon-controls'
    '@storybook/addon-backgrounds'
    '@storybook/addon-toolbars'
    '@storybook/addon-measure'
    '@storybook/addon-outline'

This change was done because addon-docs had to be reconfigured to display the full story in Canvas 'Story'.